sonar-dotnet-shared-library/src/main/java/org/sonarsource/dotnet/shared/plugins/AnalysisWarningsSensor.java

```diff
@@ -41,31 +42,29 @@
 import java.util.regex.Pattern;
 import java.util.stream.Stream;
 
-public class AnalysisWarningsSensor implements Sensor {
+@Phase(name = Phase.Name.POST)
```

Comment:
Why do we need the `POST` phase? Nothing should depend on this.

Reply:
Seems to me that none of the others have, and I would say this is not special by any means. It can be run in whatever order as part of the end step.